omx/gstomxaudiodec.c

```diff
@@ -397,12 +375,12 @@ gst_omx_audio_dec_loop (GstOMXAudioDec * self)
         default:
           /* This will break the outer loop too as the
            * i == pcm_param.nChannels afterwards */
-          for (i = 0; i < pcm_param.nChannels; i++)
+          for (i = 0; i < self->pcm_param.nChannels; i++)
```

Comment:
To me, it doesn't make sense to invalidate all the positions if a single one of them is invalid. Should we leave only the `i` position as `GST_AUDIO_CHANNEL_POSITION_NONE`?

Reply:
I agree with you. 37e7289e350de4489759ea0bee787ef86819ab60